test.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import argparse
+import functools
+import os
+import platform
+import subprocess
+import time
+
+from build import build
+
+# Disable buffered output so that the log statements and subprocess output get interleaved in proper order
+print = functools.partial(print, flush=True)
+
+parser = argparse.ArgumentParser(
+    description="Runs integration tests for the repo.",
+)
+
+parser.add_argument(
+    "--build",
+    action=argparse.BooleanOptionalAction,
+    default=True,
+    help="Build the repo first (default is --build). When this option is used, "
+    "any unparsed command line arguments are passed directly to build.py."
+)
+
+parser.add_argument(
+    "build_args",
+    nargs="*",
+    help="When --build is used, arguments passed to build.py. Example: `test.py --build -- --debug`"
+)
+
+args = parser.parse_args()
+
+def step_start(description):
```

Comment:
Being that these are being duplicated in ./build.py as well, maybe import one from the other, or just de-duplicate the scripts into one as per the later comment.